src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="conteiner">
+      <div class="calendar">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
you don't need calendar__day--#
use :nth-child() for adding styles and modifications